cli/CmdRunner.go

```diff
@@ -406,7 +406,7 @@ func (rt *CmdRunner) getNode(sim *simulation.Simulation, sel NodeSelector) (*sim
 		return sim.Nodes()[sel.Id], sim.Dispatcher().Nodes()[sel.Id]
 	}
 
-	panic("node selector not implemented")
+	return nil, nil
```

Comment:
Should we print a warning?

Reply:
Not needed in this case, because the caller will always check for 'nil' and print a message that the node can't be found. Similar to the case that a too-high node number is chosen. The message will say "node 0 not found", "src node not found", "dst node not found" etc depending on the command.

Reply:
Thanks for explaining. 

The original panic was to detect new NodeSelector other than by NodeID, that is added but not implemented.

Should be fine to just return nil.